apps/studio/src/lib/export/formats/json.ts

```diff
@@ -16,7 +16,7 @@ export class JsonExporter extends Export {
 
   constructor(
     filePath: string,
-    connection: DBConnection,
+    connection: BasicDatabaseClient<any>,
```

Comment:
I think you mean to use `DatabaseClient` here, `BasicDatabaseClient` is an abstract implementation, but there could be others.

Reply:
We don't have DatabaseClient anymore